docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -0,0 +1,170 @@
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+## Installing SpECTRE on Apple Silicon {#installation_on_apple_silicon}
+
+The following instructions show how to install SpECTRE and its dependencies
+on Apple Silicon Macs. Apple Silicon is an arm64 architecture, in contrast
+to the x86-64 architecture that SpECTRE usually targets. These instructions
+will result in an Apple Silicon native build of SpECTRE.
+
+\note Floating-point exception trapping and python bindings are not currently
+supported on Apple Silicon.
+
+### 0. Install the xcode command-line tools.
+
+Install the xcode command-line tools, which include the clang compiler, etc.
+
+~~~
+xcode-select --install
+~~~
+
+### 1. Make a directory to install prerequisites
+
+First, make a directory to hold some prerequisites that spectre depends on.
+Name this directory whatever you like, e.g. `$HOME/apps`.
+~~~
+export SPECTRE_DEPS_ROOT=$HOME/apps
+mkdir $SPECTRE_DEPS_ROOT
+cd $SPECTRE_DEPS_ROOT
+mkdir src
+cd src
+~~~
+
+### 2. Install python dependencies
+
+Spectre depends on python and some python packages. There are different ways to
+install an arm64-native python stack. The following instructions show how
+to do this using [Miniforge](https://github.com/conda-forge/miniforge),
+which supports Apple Silicon natively.
+
+#### Miniforge
+Distribution Miniforge is a way to install an arm64-native python stack on Apple
+Silicon Macs. Download the
+[Miniforge3-MacOSX-arm64.sh](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh)
+installation script, and then do the following:
+~~~
+# Install miniforge. Accept all default choices
+bash Miniforge3-MacOSX-arm64.sh
+# Activate conda base environment for current session
+eval "$(${HOME}/miniforge3/bin/conda shell.zsh hook)"
+# Activate conda at startup (to make sure you're always using native python)
+conda init zsh
+exit # close shell. conda will automatically load when new zsh shells start.
+~~~
+
+Open a new terminal window. Then, install the necessary python packages.
+~~~
+conda install numpy scipy matplotlib h5py pyyaml
+~~~
+
+You might also wish to install jupyter notebook support, using
+~~~
+conda install -c conda-forge jupyterlab
+~~~
+
+### 3. Install dependencies with Homebrew
+
+Most of spectre's dependencies beyond python can be installed using the
+[homebrew](https://brew.sh) package manager. First, if you haven't
+already, install Homebrew by
+following the instructions on the [homebrew](https://brew.sh) homepage. Then,
+run the following to install a fortran compiler and other dependencies:
+~~~
+brew install gcc
+brew install openblas boost gsl cmake doxygen
+brew install ccache autoconf automake catch2 jemalloc hdf5 pybind11 yaml-cpp
+~~~
+
+### 4. Install remaining dependencies
+
+Here, install the remaining dependencies that cannot be installed
+with homebrew or miniforge.
+
+Create a directory (such as `$HOME/apps`) to store the build dependencies.
```

Comment:
Rendering fails from the `$HOME`  on so I cannot review...

Reply:
This renders for me, but I reworded to avoid the $ inside the backquotes. Does it now render for you?